M/MDAL/build_tarballs.jl

```diff
@@ -0,0 +1,68 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MDAL"
+version = v"0.8.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/lutraconsulting/MDAL/archive/refs/tags/release-$version.tar.gz", "0051495aff910b05d04efa792925cce377920ee02a03a89fa45529fd96cd2953"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+cd $WORKSPACE/srcdir/MDAL-*
+
+atomic_patch -p1 ${WORKSPACE}/srcdir/patches/rename-findsqlite-library.patch
+
+CMAKE_FLAGS=(-DCMAKE_INSTALL_PREFIX=$prefix
+-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}
+-DCMAKE_BUILD_TYPE=Release
+-DENABLE_TESTS=OFF
+-DENABLE_COVERAGE=OFF
+-DWITH_HDF5=OFF
+-DWITH_GDAL=ON
+-DWITH_XML=ON
+-DWITH_SQLITE3=ON
+-DBUILD_STATIC=OFF
+-DBUILD_SHARED=ON
+-DBUILD_TOOLS=OFF
+-DBUILD_EXTERNAL_DRIVERS=OFF)
+
+#NetCDF is the most restrictive dependency as far as platform availability, so we'll use it where applicable but disable it otherwise
+if [-f ${libdir}/libnetcdf*.${dlext}]; then
```

Comment:
I don't think this worked, let's try this instead
```suggestion
if [[ -f ${libdir}/libnetcdf*.${dlext} ]]; then
```